### DISCUSSION TITLE
Sound argument

### DISCUSSION THREAD

**Discussion by KyllianGamer** [Minestom/Minestom#732] [Help] [answered]
How can I make an argument out of the SoundEvents interface? (It has all the vanilla Sounds)
-> For command use

**Comment by MrlnHi** [comment, marked as answer]
I have looked around for a little now, and I couldn't find a clean way. Here is an ugly way that should work (but if someone knows a better way please provide it)
```java
ArgumentWord argSound = ArgumentType.Word("sound")
                .from(SoundEvent.values().stream().map(SoundEvent::namespace).map(NamespaceID::toString).toArray(String[]::new));
```

**Reply by KyllianGamer** [reply]
Works perfectly fine, thank you.
